types/index.d.ts

```diff
@@ -1243,6 +1243,14 @@ declare namespace RamdaAdjunct {
          */
         allUnique<T>(list: T[]): boolean;
 
+        /**
```

Comment:
The description should exactly match the description provided in `padCharsEnd.js`.
```js
* The function pads the current string with a given string
 * (repeated, if needed) so that the resulting string reaches a given length.
 * The padding is applied from the end of the current string.